doc/sphinx/locale/es/LC_MESSAGES/external_outgoing_calls.po

```diff
@@ -147,7 +147,7 @@ msgstr "Conmutación por error"
 #: ../../external_outgoing_calls/call_routing.rst:73
 msgid "Failover route lets us use another route whenever the main route fails."
 msgstr "Las rutas en caso de fallo o *failover-routes* nos permite disponer de "
-"rutaadicional en caso de que la ruta preferida falle."
+"ruta adicional en caso de que la ruta preferida falle."
```

Comment:
I think the original translation for this one will make more sense with *otra ruta adicional* or *una ruta adicional*